relay-server/src/actors/store.rs

```diff
@@ -950,7 +950,6 @@ struct MetricKafkaMessage {
     #[serde(flatten)]
     value: BucketValue,
     timestamp: UnixTimestamp,
-    #[serde(skip_serializing_if = "BTreeMap::is_empty")]
```

Comment:
i am reviewing this change only. i think we can revisit the schema we actually want later, but as pointed out in private conversations, the metrics indexer currently does not handle the absence of `tags` correctly.